src/open_inwoner/scss/components/Contactmomenten/Contactmomenten.scss

```diff
@@ -74,46 +74,22 @@
 }
 
 .contactmoment {
-  &__details {
-    // Set table width for word-break
-    table {
-      margin-top: 0;
-      overflow-x: auto;
-      table-layout: fixed;
-      width: 100%;
-      @media (min-width: 768px) {
-        width: var(--mobile-ms-width);
-      }
-    }
-    table th {
-      width: 100px;
-      white-space: nowrap;
-    }
-    table td {
-      width: 300px;
-      overflow: hidden;
-      word-break: break-word;
-      white-space: normal;
-
-      &.table__item--status {
-        .table__item--wrapper {
-          display: flex;
-          justify-content: space-between;
-        }
-      }
+  &__heading {
+    font-family: var(--font-family-heading);
+    font-size: var(--font-size-heading-3);
+    font-weight: var(--font-weight-heading-3);
+    line-height: var(--font-line-height-heading-3);
+    margin: 0;
+  }
 
-      .case-detail__link {
-        color: var(--color-primary);
-        text-decoration: none;
+  &__text {
```

Comment:
Since you're not using the "CSS Reset" styling you will need to style **_everything_** for paragraphs here. A "CSS reset" means you're setting paddings and margins to zero, because many HTML elements have a default padding/margin you'll need to 'reset/null' in order to give it the proper values from the actual design.

TODO: 
- remove the padding from these '&__text' paragraphs
- add margin: 0 0 var(--row-height) 0;

Note: --row-height=40px.

It would also be slightly more proper to add the font-family=body here, though it's already set on the Body tag.

Look at all the stuff that the `.p` class is 'resetting'; these are the values you will need every time you want to give a paragraph the proper styling if you do not wish to use the `.p` class (and yeah: we are slowly trying to get rid of that one) so this is how it is 'resetting' default HTML values:

```
.p {
  color: var(--font-color-body);
  font-family: var(--font-family-body);
  font-size: var(--font-size-body);
  line-height: var(--font-line-height-body);
  font-weight: normal;
  margin: 0;
}
```